### PR TITLE
fixed visible to participant bug on casenote form

### PR DESCRIPTION
**Pull request by erinysong**
## Fixed Casenote Internal Update Bug

Internal variable on casenotes were originally not properly updating if you set the casenote to non-internal - fixed by having the internal state var take in props instead of setting it as a default value
 
### Related PRs

Optional - any related PRs you're waiting on, or PRs that will conflict, etc

### Migrations

Optional - if you added anything to the database through migration(s)

### Tests Performed, Edge Cases

Lol we'll see if we test

### Screenshots

Add screenshots!!! If you'd like

CC: @didvi

### CODE REVIEW

File: app/javascript/components/CaseNoteForm.jsx [hunk 1]
```diff
@@ -128,7 +128,12 @@ class CaseNoteForm extends React.Component {
   };
 
   handleInternalChange = name => event => {
-    this.setState({ [name]: !this.state.internal });
+    console.log("before " + this.state.internal);
```
Comment (wilsonw926) [inline app/javascript/components/CaseNoteForm.jsx]: Remove the console.log statements unless we need them in the application.

File: app/javascript/components/CaseNoteForm.jsx [hunk 2]
```diff
@@ -128,7 +128,12 @@ class CaseNoteForm extends React.Component {
...
+    console.log("target " + event.target.checked);
+    this.setState(
+      { [name]: event.target.checked },
+      () => console.log('now', this.state.internal)
```
Comment (wilsonw926) [inline app/javascript/components/CaseNoteForm.jsx]: same with this console.log

File: app/javascript/components/CaseNoteForm.jsx [hunk 3]
```diff
@@ -177,6 +183,8 @@ class CaseNoteForm extends React.Component {
           participant_id: this.state.participant_id,
         };
 
+        console.log(body);
```
Comment (wilsonw926) [inline app/javascript/components/CaseNoteForm.jsx]: remove console.log
